CMakeLists.txt

```diff
@@ -13,21 +17,19 @@ find_package(Eigen3 3.3 REQUIRED NO_MODULE)
 find_package(GTest REQUIRED)
 find_package(absl REQUIRED)
 find_package(Ceres REQUIRED)
-find_package(pybind11 REQUIRED)
-find_package(Python REQUIRED)
 find_package(OpenCV REQUIRED)
 find_package(yaml-cpp REQUIRED)
 
 # Add top level include directory.
 include_directories(
-  ${CMAKE_CURRENT_SOURCE_DIR}
+  ${CMAKE_CURRENT_LIST_DIR}
```

Comment:
this change just has CMake use the _current_ file's location as the reference point instead of the origin file - there's no difference  when this is the top-level file, but if its imported this keeps the paths relative to this file.